### PR TITLE
 secrets vs configmap

### PR DESCRIPTION
**Pull request by cooktheryan**

### CODE REVIEW

**Comment by cooktheryan** [comment]
CTLog and TUF still remain

File: roles/sigstore_scaffolding/tasks/podman/keycloak.yml [hunk 1]
```diff
@@ -37,23 +37,26 @@
     - "{{ certs_dir }}/ingress-keycloak.key"
   register: remote_keycloak_certificates
 
+- name: base64 encode the keycloak ingress
+  shell: echo "{{ (remote_ingress_certificates.results | selectattr('source', 'equalto', certs_dir + '/ingress-keycloak.pem') | list | first).content | b64decode + (remote_ingress_certificates.results | selectattr('source', 'equalto', remote_sigstore_ca) | list | first).content | b64decode }}" | base64
```
Comment (cooktheryan) [inline roles/sigstore_scaffolding/tasks/podman/keycloak.yml]: @sabre1041 @sallyom looking for advice here. This is ugly but it works. The problem exists specifically with the pem files where we need to decode to be able to have one big blob containing both the pem and the ca then encrypt the whole blob. If we don't encode it as a blob it causes an issue when trying to create the certificate

Comment (sallyom) [reply roles/sigstore_scaffolding/tasks/podman/keycloak.yml]: I don't have a better idea atm.

Comment (sabre1041) [reply roles/sigstore_scaffolding/tasks/podman/keycloak.yml]: Is there a reason we are using the `shell` module? We should be able to use `set_fact` to achieve the same result

File: roles/sigstore_scaffolding/tasks/podman/nginx.yml [hunk 2]
```diff
@@ -31,39 +31,50 @@
     - "{{ certs_dir }}/ingress-keycloak.key"
   register: remote_ingress_certificates
 
+- name: base64 encode the rekor ingress
```
Comment (sabre1041) [inline roles/sigstore_scaffolding/tasks/podman/nginx.yml:34]: All of these statements can be combined into a series of `set_fact`

**Review by sallyom — APPROVED** [review]
LGTM!

**Pull request merged**